api/views.py

```diff
@@ -41,3 +49,28 @@ def get(self, request, format=None):
                                   .prefetch_related('person_set')
         serializer = ExportInstructorLocationsSerializer(airports, many=True)
         return Response(serializer.data)
+
+
+class ListEvents(APIView):
+    # I wanted to use ListAPIView, but it had problems with the way we test
+    # this code...
```

Comment:
The ListAPIView used pagination and with our fake request in tests (essentially `request = None`) it was throwing AttributeErrors (due to the fact that request was None and the ListAPIView class used pagination). Switching to APIView got rid of pagination and therefore we can fake requests in tests successfully.